app/src/main/java/com/github/libretube/helpers/WindowHelper.kt

```diff
@@ -20,25 +21,10 @@ object WindowHelper {
 
         WindowCompat.setDecorFitsSystemWindows(window, !isFullscreen)
 
-        val controller = WindowCompat.getInsetsController(window, window.decorView)
-        val flags = WindowInsetsCompat.Type.systemBars() or WindowInsetsCompat.Type.navigationBars()
         if (isFullscreen) {
-            controller.hide(flags)
+            activity.hideSystemBars(WindowInsetsCompat.Type.systemBars())
```

Comment:
Due to #3069, hiding the navigation bars here too might fix issues for some devices.

Reply:
![Screenshot_20230401-122249_MX_Player_Pro](https://user-images.githubusercontent.com/66936649/229270692-e061bb9e-75f0-42e5-8b18-8c8ea9478bb1.png)

Do want navigation bars this way or not appear when controls are visible

Reply:
They should not appear, but if the user manually shows them they should be hidden again automatically.